go/vt/vtgate/planbuilder/show.go

```diff
@@ -265,7 +268,17 @@ func buildShowVMigrationsPlan(show *sqlparser.ShowBasic, vschema plancontext.VSc
 		dest = key.DestinationAllShards{}
 	}
 
-	sql := "SELECT * FROM _vt.schema_migrations"
+	sdbidc, err := sidecardb.GetIdentifierCache()
+	if err != nil {
+		return nil, err
+	}
+	sidecarDBID, err := sdbidc.Get(ks.Name)
+	if err != nil {
+		return nil, vterrors.Wrapf(err, "failed to read sidecar database identifier for keyspace %q from the cache",
+			ks.Name)
+	}
```

Comment:
in vtgate planner or executor, we through errors with an error code
example: `vterrors.VT13001`